includes/class-newspack-popups-model.php

```diff
@@ -1167,7 +1167,7 @@ class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>"
 			id="<?php echo esc_attr( $element_id ); ?>"
 		>
 			<div class="<?php echo esc_attr( implode( ' ', $wrapper_classes ) ); ?>" data-popup-status="<?php echo esc_attr( $popup['status'] ); ?>" style="<?php echo ! $hide_border ? esc_attr( self::container_style( $popup ) ) : ''; ?>">
-				<div class="newspack-popup" style="<?php echo $hide_border ? esc_attr( self::container_style( $popup ) ) : ''; ?>">
```

Comment:
There are a few places where a nested `.newspack-popup` is used in CSS selector, e.g.:

- [Theme patterns 7](https://github.com/Automattic/newspack-theme/blob/4ae87e805c542ef88be957fd8c426fadfe079d90/newspack-theme/sass/blocks/_patterns.scss#L68-L69) & 8
- [This plugin, in sizing handling](https://github.com/Automattic/newspack-popups/blob/72474694a59b6502a052d45351c13e15e118b3bf/src/view/style.scss#L224-L226)

Reply:
Ah, I see that it's actually used for styling. 868e742 changes the name to `.newspack-popup__content-wrapper` instead to avoid the naming collision while retaining the styles, and